R/ResampleDesc.R

```diff
@@ -64,6 +64,9 @@
 #'     else it will be a fraction of the total training indices. IE for 100 training sets and a value of .2, the increment
 #'     of the resampling indices will be 20. Default is \dQuote{horizon} which gives mutually exclusive chunks
 #'      of test indices.}
+#'   \item{fixed (`logical(1)`)}{Whether indices supplied via argument 'blocking' in the task should be used in resampling. Default is `FALSE`.
```

Comment:
This wording suggests that blocking is ignored in resampling, and the documentation for blocking says the opposite.

Reply:
Made it more clear now. Please take another look :)